joomla/joomla-cms
# [4.2] Clean the logo URL in Cassiopeia

Pull Request for Issue #39571, https://github.com/joomla/joomla-cms/issues/39538, https://github.com/joomla/joomla-cms/issues/39543

### Summary of Changes
Use the recommended way to render an image: through the HTMLHelper


### Testing Instructions
Check that selecting a custom logo image works and that the source doesn't have the hash part of the URL. Also check that svg renders as expected


### Actual result BEFORE applying this Pull Request

The URL contains the hash (it is dirty) and SVG images get 0 width and height thus are not visible

### Expected result AFTER applying this Pull Request

The URL **does not** contain the hash (it is clean) and SVG images are visible


### Link to documentations
This is a bug fix without changing any of the existing functionality
Please select:
